pkg/cgroups/cgroups.go

```diff
@@ -38,6 +38,9 @@ var (
 
 	// Only mount a single instance
 	cgrpMountOnce sync.Once
+
+	// Regex to match 'kubelet'
+	re = regexp.MustCompile(`^.*kubelet`)
```

Comment:
Can we have a more specific name for this, for instance `kubeletRegex`?

Reply:
Done. PTAL